Code/VideoPlayerKit.h

```diff
@@ -25,5 +25,8 @@
 + (VideoPlayerKit *)videoPlayerWithContainingViewController:(UIViewController *)containingViewController
```

Comment:
I think we're good with removing this option. I like the containingView way

Reply:
I've added a deprecate attribute to the old videoPlayerWithContainingViewController instead of removing it so far to alert developers to use the new UIView method. Take a look and let me know if you'd like me to keep it as is or removing it as suggested!

Reply:
nice!